src/bench/bench_bitcoin.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "bench.h"
+
+#include "chainparams.h"
+#include "key.h"
+#include "main.h"
+#include "random.h"
+#include "txdb.h"
+#include "ui_interface.h"
+#include "util.h"
+
+#include <boost/filesystem.hpp>
+
+extern bool fPrintToConsole;
+extern void noui_connect();
+
+int
+main(int argc, char** argv)
+{
+    ECC_Start();
```

Comment:
`ECC_*()` are here probably because you are testing `ConnectBlock()`? This should be in the specific benchmark init section IMO, not here.